tests/spread/integration/libxml-utils/task.yaml

```diff
@@ -0,0 +1,15 @@
+summary: Integration tests for libxml2-utils
```

Comment:
The directory name for this should be `libxml2-utils` instead of `libxml-utils`.